mobile/pubspec.lock

```diff
@@ -61,10 +61,10 @@ packages:
     dependency: transitive
     description:
       name: archive
-      sha256: ed7cc591a948744994714375caf9a2ce89e1d82e8243997c8a2994d57181c212
+      sha256: d6347d54a2d8028e0437e3c099f66fdb8ae02c4720c1e7534c9f24c10351f85d
```

Comment:
@NoahNsimbe, I've made these changes `flutter pub upgrade` because the build was originally failing.

Reply:
No worries @Mnoble-19.  This is fine. It won't break anything. LGTM